examples/airline/configs/agents.py

```diff
@@ -0,0 +1,93 @@
+from configs.tools import *
+from data.routines.baggage.policies import *
+from data.routines.flight_modification.policies import *
+from data.routines.prompts import STARTER_PROMPT
+
+from openai_runner import Agent
+
+
+def transfer_to_flight_modification():
+    """Call this function when a user needs to be transferred to flight modification agent."""
+    return flight_modification
+
+
+def transfer_to_flight_cancel():
+    return flight_cancel
+
+
+def transfer_to_flight_change():
+    return flight_change
+
+
+def transfer_to_lost_baggage():
+    return lost_baggage
+
+
+def transfer_to_triage():
+    """Call this function when a user needs to be transferred to a different agent and a different policy.
+    For instance, if a user is asking about a topic that is not handled by the current agent, call this function.
+    """
+    return triage_agent
+
+
+def triage_instructions(context_variables):
+    customer_context = context_variables.get("customer_context", None)
+    flight_context = context_variables.get("flight_context", None)
+    return f"""You are to triage a users request, and call a tool to transfer to the right intent.
+    Once you are ready to transfer to the right intent, call the tool to transfer to the right intent.
+    You dont need to know specifics, just the topic of the request.
+    When you need more information to triage the request to an agent, ask a direct question without explaining why you're asking it.
+    Do not share your thought process with the user! Do not make unreasonable assumptions on behalf of user.
+    The customer context is here: {customer_context}, and flight context is here: {flight_context}"""
```

Comment:
**Suggestion:** Modify the `triage_instructions` to ensure the string formatting does not introduce unnecessary newlines and spaces, which could affect the readability and presentation of the context in the UI. [general, importance: 4]
```suggestion
    return f"You are to triage a user's request. Once ready, use the appropriate tool to transfer to the correct intent. \
    Do not share your thought process with the user! The customer context is: {customer_context}, and flight context is: {flight_context}"
```